src/acceptance/api/api_test.go

```diff
@@ -104,7 +104,7 @@ var _ = Describe("AutoScaler Public API", func() {
 			resp, err := DoAPIRequest(req)
 			Expect(err).ShouldNot(HaveOccurred())
 			defer resp.Body.Close()
-			Expect(resp.StatusCode == 200 || resp.StatusCode == 404).Should(BeTrue())
+			Expect([]int{200, 404}).To(ContainElement(resp.StatusCode))
```

Comment:
this is weird.. you can use 
Or(matchers …GomegaMatcher)
SatisfyAny(matchers …GomegaMatcher)